gnomad_qc/v4/plots/sample_growth.R

```diff
@@ -0,0 +1,78 @@
+# R script to get the sample growth of gnomAD v2, v3, and v4 exomes and genomes.
+# To run this script, simply copy and paste the code into an R console or use `Rscript sample_growth.R` in the terminal.
+
+# change the working directory as needed, note: the folder should already exist.
+setwd("~/Documents")
+
+if (!require("tidyr")) install.packages("tidyr")
+
+color_amr = k_amr = '#ED1E24'
+color_eur = k_eur = '#6AA5CD'
+color_afr = k_afr = '#941494'
+color_sas = k_sas = '#FF9912'
+color_eas = k_eas = '#108C44'
+color_oth = k_oth = '#ABB9B9'
+color_mde = k_mde = '#000080'
+color_asj = k_asj = 'gold2'
```

Comment:
```suggestion
color_mde = k_mde = '#33CC33'
color_asj = k_asj = '#FF7F50'
```
Updating to match gnomad_methods colors: https://github.com/broadinstitute/gnomad_methods/blob/7c0c994883f321492a48962674d5caeb289df4c7/gnomad/sample_qc/ancestry.py#L64C13-L64C20